pkg/ddevapp/templates.go

```diff
@@ -70,7 +70,7 @@ const ConfigInstructions = `
 # change from the default system Node.js version to any other version.
 # Numeric version numbers can be complete (i.e. 18.15.0) or
 # incomplete (18, 17.2, 16). 'lts' and 'latest' can be used as well along with
-# other named releases.
+# other named releases, 'auto' reads the version from a file such as '.nvmrc' or 'package.json'.
```

Comment:
In the pr description you mentioned an `engine` setting for checking `package.json`, but here you say `auto` will do that?

Reply:
May be worth mentioning the other files that it reads from (there's only 3 after all) to avoid people thinking it's only `.nvmrc`

Reply:
I didn't even notice that I copied the part with `engine` 🙂, it doesn't work now, but it will.
I'm going to remove my hardcoded condition for `auto` so that it accepts anything that works for `n`.